feat(fdb): new doc db - MTA-3490 (#2000)
scaleway/docs-content: managed-databases/document-database/quickstart.mdx

> </Message>

Reviewer: Remove this section as we are switching to Public beta

        - **High Availability**: creates a secondary node with an up-to-date replica of the database. If the primary Instance fails, the secondary takes over requests.
        - **Standalone**: creates a standalone database provisioned on a single node.
    - Select a node type.
    - Define the size of your Block Storage volume. You can increase your storage space without changing your node type, with no downtime. You can increase your volume to up to 10 TB.
    - Enable automatic backups, if necessary.
    - Add a name and set a password for your user.
    <Message type="important">
    Your username must adhere to specific criteria.
      - Length must be between 1 and 63 characters
      - First character must be an alphabetic character (**a-Za-Z**)
      - It can not start with **_rdb**
      - Only **a-zA-Z0-9_$-** characters are accepted
    </Message>
    - Enter a name for your Database Instance.
    - Tick the checkbox if you wish to share telemetry data with FerretDB.
    <Message type="important">

>       This option is available during the Public Beta phase of Document Databases. It allows you to contribute with the FerretDB Open Source project. Only request types, such as find and aggregate, will be collected and shared. Request parameters or response content is not collected nor shared. Refer to the official Ferret DB [documentation page on telemetry](https://docs.ferretdb.io/telemetry/) for more information.

Reviewer 2: ```suggestion
      This option is available during the Public Beta phase of Document Databases. It allows you to contribute with the FerretDB Open Source project. Only request types, such as find and aggregate, will be collected and shared. Request parameters or response content are neither collected nor shared. Refer to the official Ferret DB [documentation page on telemetry](https://docs.ferretdb.io/telemetry/) for more information.
```

### Via database client

1. Click **Document Databases** under **Managed Databases** on the side menu. A list of your Database Instances displays.
2. Click the database name or <Icon name="more" /> > **More info** to access the Database Instance information page.
3. Click <Icon name="download" /> next to the **TLS certificate** to download it.
4. Transfer the file to where you will execute the connection command.
5. Copy the connection command located under **Connection** in the Database Instance information page.
6. Paste the connection command in your terminal, and edit the username field with the login you chose when creating the database. The full command should look like the following:
    ```bash

>     mongosh --tls --tlsCAFile rdb-databasename.pem 'mongodb://username@ip:port/databasename?authMechanism=PLAIN'

Reviewer: since "rdb-" prefix was relevant (it has been changed to "docdb" now in the automatically suggested database instance name), it has been removed from the downloaded certificate file name.
Moreover, the certificate name is not the database name, but the "database instance" name (the issue already existed before these modifications).  I don't know if difference between database instance and database is clear to user, maybe we can emphasize that the database instance is the resource created in Console and might look like "docdb-xxxxx-xxxx" if created automatically.
```suggestion
    mongosh --tls --tlsCAFile databaseinstancename.pem 'mongodb://username@ip:port/databasename?authMechanism=PLAIN'
```

    - `ip` - your Database Instance's endpoint IP address.

>     - `database name` - the name of the database you entered upon Database Instance creation. The default is called `rdb`.

Reviewer 2: ```suggestion
    - `database name` - the database name you entered upon Database Instance creation. The default is called `rdb`.
```

>     - `database name` - the name of the database you entered upon Database Instance creation. The default is called `rdb`.

Reviewer: To be consistent with comment above:
```suggestion
    - `database instance name` - the name of the database you entered upon Database Instance creation.
    - `database name` - The default is called `rdb`. If you created a new database from the Databases tab inside your Database Instance, replace by the database name.
```

    - `port` - the port on which your Database Instance is accessible
7. Enter your password.

> 8. If the connection is successful, you should see the following message display on your console, and be able to write queries:

Reviewer 2: ```suggestion
If the connection is successful, you should see the following message display on your console, and be able to write queries:
```
